### PR TITLE
Update pypi package metadata

### PR DESCRIPTION
**Pull request by faxu**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -324,33 +324,42 @@ def run(self):
...
         'Operating System :: MacOS',
+        'Topic :: Scientific/Engineering',
+        'Topic :: Scientific/Engineering :: Mathematics',
+        'Topic :: Scientific/Engineering :: Artificial Intelligence',
+        'Topic :: Software Development',
+        'Topic :: Software Development :: Libraries',
+        'Topic :: Software Development :: Libraries :: Python Modules',
         'Programming Language :: Python',
         'Programming Language :: Python :: 3 :: Only',
         'Programming Language :: Python :: 3.5',
```
Comment (xadupre) [inline setup.py:361]: #6235 removed Python 3.5 from CI. Maybe we should remove it from here too.

Comment (faxu) [reply setup.py]: thanks. removed it. @snnn is 3.9 ready to be added yet?

File: setup.py [hunk 2]
```diff
@@ -324,33 +324,41 @@ def run(self):
...
+        'Topic :: Scientific/Engineering :: Mathematics',
+        'Topic :: Scientific/Engineering :: Artificial Intelligence',
+        'Topic :: Software Development',
+        'Topic :: Software Development :: Libraries',
+        'Topic :: Software Development :: Libraries :: Python Modules',
         'Programming Language :: Python',
         'Programming Language :: Python :: 3 :: Only',
-        'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7'],
```
Comment (xadupre) [inline setup.py:362]: You need to remove the final `]` on this line. That's why the build is failing.

Comment (faxu) [reply setup.py]: whoops. thank you, fixed.

**Review by snnn — APPROVED** [review]

**Pull request merged**